src/lib/y2storage/planned/lvm_lv.rb

```diff
@@ -124,6 +163,25 @@ def volume_match_values
           partition_id: nil
         }
       end
+
+      # Returns the size for the LV in a given volume group/thin pool when specified as percentage
+      #
+      # @param container [LvmVg,LvmLv] Volume group or thin pool where the logical volume will
+      #   be placed
+      # @return [DiskSize]
+      def size_in_percentage(container)
+        extent_size = container.is?(:lvm_lv) ? container.lvm_vg.extent_size : container.extent_size
+        (container.size * percent_size / 100).floor(extent_size)
+      end
+
+      # Returns the size for the logical volume in a given thin pool
+      #
+      # @param container [LvmVg,LvmLv] Volume group or thin pool where the logical volume will
+      #   be placed
+      # @return [DiskSize]
+      def size_in_thin_pool(container)
```

Comment:
NP: I guess the parameter will always be a `thin_pool`. Otherwise, the method name would be weird.